dlc-oracle/src/main/scala/org/bitcoins/dlc/oracle/config/DLCOracleAppConfig.scala

```diff
@@ -44,17 +45,37 @@ case class DLCOracleAppConfig(
   }
 
   override def start(): Future[Unit] = {
-    logger.debug(s"Initializing wallet setup")
-    for {
-      _ <- super.start()
-    } yield {
+    logger.debug(s"Initializing dlc oracle setup")
+    super.start().flatMap { _ =>
       if (Files.notExists(datadir)) {
         Files.createDirectories(datadir)
       }
       val numMigrations = {
         migrate()
       }
       logger.info(s"Applied $numMigrations to the dlc oracle project")
+
+      if (migrationsApplied() == 2) {
```

Comment:
nit: Maybe add a comment why this is necessary

Reply:
Isn't this the wrong ordering? 

This seems to do 

1. Migrate tables
2. Read existing data from database
3. Insert data into new tables

but don't we want this ordering:

1. Read existing data from database
2. Apply migration
3. Insert data into new tables

Reply:
This is correct. We need to the migrations first so we can read the data, otherwise we would need the old DAOs.

Reply:
What the migrations do is put in a dummy event descriptor. Then here we calculate the correct event descriptor and update it